.travis/yapf.sh

```diff
@@ -19,24 +19,23 @@ git fetch upstream master
 
 YAPF_FLAGS=(
     '--style' "$ROOT/.style.yapf"
-    '--in-place'
     '--recursive'
     '--parallel'
 )
 
 YAPF_EXCLUDES=(
-    '--exclude' 'python/ray/dataframe'
-    '--exclude' 'python/ray/rllib'
-    '--exclude' 'python/ray/cloudpickle'
-    '--exclude' 'python/build'
-    '--exclude' 'python/ray/pyarrow_files'
-    '--exclude' 'python/ray/core/src/ray/gcs'
-    '--exclude' 'python/ray/common/thirdparty'
+    '--exclude' 'python/ray/dataframe/*'
+    '--exclude' 'python/ray/rllib/*'
+    '--exclude' 'python/ray/cloudpickle/*'
+    '--exclude' 'python/build/*'
```

Comment:
The '/*' are pointless because listing the name of the directory causes yapf to exclude the whole thing since the --recursive flag is passed in YAPF_FLAGS.

Reply:
Unless you specify the star, the excludes have no effect. Try this yourself. They yapf docs also mention the exclude is supposed to be a *pattern*.

Reply:
I just ran the script with `-all` and added the `--verbose` to see the files processed. It seems to work just fine without the pattern, skipping the correct files.